steembase/types.py

```diff
@@ -262,15 +264,15 @@ def __str__(self):
 
 class Bool(Uint8):  # Bool = Uint8
     def __init__(self, d):
-        super().__init__(d)
+        Uint8.__init__(self, d)
```

Comment:
Not sure if there is a difference here but I notice that the pattern you used elsewhere was in the form of `super(Uint8, self)`

Reply:
Some examples of those classes are in `operations.py`. Those are "new-style" classes that inherit from `object`, where as here, Uint8 is an "old-style" class. This is the syntax for calling the super on an old-style class.

Reply:
:+1: